src/modules/endpoint/endpoints/extras.js

```diff
@@ -4,14 +4,14 @@ import { apiUrl } from '../constants'
 
 export default new EndpointBuilder('extras')
   .withStateToUrlMapper(state => `${apiUrl}/${state.router.params.location}` +
-    `/${state.router.params.language}/wp-json/extensions/v3/extras/`)
+    `/${state.router.params.language}/wp-json/extensions/v3/extras`)
   .withMapper(json => json
-    .filter(extra => extra.enabled && extra.alias !== 'ige-c4r')
+    .filter(extra => extra.alias !== 'ige-c4r')
```

Comment:
is this still relevant? Do we still receive careers4refugees extras from the backend?

Reply:
I asked timo, not relevant anymore